ExadelTimeTrackingSystem/ExadelTimeTrackingSystem.WebAPI/Controllers/TasksController.cs

```diff
@@ -97,8 +102,35 @@ public async Task<ActionResult<List<BulkCreateTaskDTO>>> BulkCreateAsync([FromBo
         {
             var cancellationToken = CancellationTokenCreator.Create(_options.CurrentValue.TimeOutSeconds);
             cancellationToken.ThrowIfCancellationRequested();
-            var tasksDto = await _service.BulkCreateAsync(bulkCreateTaskDto, cancellationToken);
+            var tasksDto = await _taskService.BulkCreateAsync(bulkCreateTaskDto, cancellationToken);
             return Created(string.Empty, tasksDto);
         }
+
+        [HttpPost("emailtest")]
```

Comment:
you should create endpoint for approving  employee tasks for week. In this endpoint tasks statuses should be updated and emails to approvers should be sent.
Endpoint input: start date of week with corresponding validation